apps/judicial-system/api/src/app/modules/defendant/models/subpoena.model.ts

```diff
@@ -0,0 +1,31 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class Subpoena {
+  @Field(() => ID)
+  readonly id!: string
+
+  @Field(() => String, { nullable: true })
+  created?: string
+
+  @Field(() => String, { nullable: true })
+  modified?: string
+
+  @Field(() => String, { nullable: true })
```

Comment:
Like, the bunny said, should this not be a `ID`?

I'm changing `acknowledged` to be `ServiceStatus` in my PR so you don't need to change that here.

Reply:
Already changed acknowledge, both in the bakend and api models. It was wrongly typed `stirng` in both places.

Regarding the ID type, I agree that generally we should use ID for unique id's. However, I am not sure about this particular field as it is not referring to an object in our system, but an external objexct in the police system. I will leave it as is, but don't really have a strong opinion.